webapp/sources/scala-ldap/src/main/scala/com/normation/ldap/sdk/LDAPConnectionProvider.scala

```diff
@@ -299,11 +309,16 @@ class RWAnonymousConnectionProvider(
  * connection provider
  */
 class ROPooledAnonymousConnectionProvider(
-    override val host:           String = "localhost",
-    override val port:           Int = 389,
-    override val ldifFileLogger: LDIFFileLogger = new DefaultLDIFFileLogger(),
-    override val useSchemaInfos: Boolean = false,
-    override val poolSize:       Int = 2
+    override val host:                           String = "localhost",
+    override val port:                           Int = 389,
+    override val ldifFileLogger:                 LDIFFileLogger = new DefaultLDIFFileLogger(),
+    override val useSchemaInfos:                 Boolean = false,
+    override val poolSize:                       Int = 2,
+    override val createIfNecessary:              Boolean = true,
+    override val minimumAvailableConnectionGoal: Int = 2,
+    override val maxWaitTime:                    Duration = 30.seconds,
```

Comment:
the default values are needed since we already have parameters with default values before them.